scripts/scil_correlate_qc_dwi.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+"""
+
+import argparse
+
+from dipy.io.gradients import read_bvals_bvecs
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.utils import (assert_inputs_exist,
+                             add_force_b0_arg)
+from scilpy.utils.bvec_bval_tools import check_b0_threshold, normalize_bvecs
+import math
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
```

Comment:
Can you please add a help about what it is doing please:
"This script simplys find your 3 closest angular neighbor (per shell) and compute the voxel-wise correlation, if it is below the average (by 2 std) it will flag the volume and print it."

Reply:
Specify if this script is robust to multi-shell data.